commands/voiceChannelManager.js

```diff
@@ -21,6 +21,14 @@ for (const [name, emoji] of Object.entries(EMOJIS)) {
   }
 }
 
+function removeDelineators(input) {
+  if (input.includes('->') || input.includes('➡️')) {
+    input = input.replace(/->|➡️/g, '');
+    return removeDelineators(input);
+  }
+  return input;
```

Comment:
"Keep replacing until it stops changing". This will fix the problem with separators in separators.

```suggestion
  while (input.includes('->') || input.includes('➡️')) {
    input = input.replace(/->|➡️/g, '');
  }
  return input;
```

Reply:
And this is why programming after the point of being tired is a bad idea lol good call